data/distrodefs/rhel-10/imagetypes.yaml

```diff
@@ -821,6 +821,28 @@
       - "customizations.timezone"
       - "customizations.rhsm"
 
+    # supported options for container types
+    supported_options_container: &supported_options_container
+      - "distro"
+      - "packages"
+      - "modules"
+      - "groups"
+      - "enabled_modules"
+      - "minimal"
+      - "containers"
+      - "customizations.directories"
+      - "customizations.files"
+      - "customizations.firewall"
```

Comment:
Sorry, this is a bit unrelated and fine of course but I wonder if people configure firewalls in their containers. No need to change anything, I just stumbled upon it and was wondering :)